tests/test_benchmarking/test_data/test_simulated_data/test_mock_student_provider.py

```diff
@@ -344,21 +344,29 @@ def test_attribute_values_from_range__multiple_values_for_attribute_are_selected
             # FIXME: This test is (technically) a little flaky 😬
             # run this multiple times so that this test cannot accidentally
             #   pass even if the values are selected with replacement
-            values_1 = attribute_values_from_range([10, 20, 30], num_values=3)
+            values_1 = attribute_values_from_range(
```

Comment:
I think you definitely gotta add more test cases to be sure this new addition works